app/models/data/average_pass_scores.rb

```diff
@@ -0,0 +1,25 @@
+module Data
+  class AveragePassScores
+    include ToCsv
+
+    # @return [Array<String>]
+    def self.column_names
+      ['Module', 'Average Pass Score']
+    end
+
+    # @return [Array<Array<String>>]
+    def self.dashboard
+      average_pass_scores.map { |module_name, average_score| [module_name, average_score] }
+    end
+
+    # @return [String]
+    def self.to_csv
+      generate_csv
+    end
+
+    # @return [ActiveRecord::Result]
+    def self.average_pass_scores
+      UserAssessment.summative.passes.group(:module).average('CAST(score AS float)')
```

Comment:
@jack-coggin can you please add a Rails `TODO:` note or a Yard `@note` for us here as a reminder. An upcoming piece of work moving from `UserAnswer` to `Response` will include changes to `UserAssessment` and we can include a type change to float so coercion is not necessary.

Reply:
Sure [03c1e40](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/648/commits/03c1e404a996e9d450c29b4b58b2e26240aff95c)